javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+const $h1 = document.querySelector('h1');
+
+const nIntervId = setInterval(function () {
+  let counter = $h1.textContent;
```

Comment:
Rather than grabbing the `textContent` from the DOM, why not utilize a counter variable?